### ISSUE TITLE
tidy.marssMLE additions

### ISSUE THREAD

**Issue by eeholmes** [nwfsc-timeseries/MARSS#27]
This issue is to document where I am at on the tidy branch.

**Comment by eeholmes** [comment]
work on tidy.marssMLE.Rd

* Done - added section on parameters vs states. For states, changed to quantiles versus conf.intervals, sd versus se.
* Partially done - similar section on observations.

To Do

**States**

This reworking is partially done, but need to revisit with fresh eyes.

* Rework the note at bottom. Shown interval is sd of X(t)|Y(1)=y(1) not se of E[X(t)|Y(1)].  Need Fisher Information matrix in latter since that captures the se of the parameters used to compute E[X(t)].  The se of E[X(t)|Y(1)] is going to be narrower than sd of X(t)|Y(1)=y(1). Just like in a Normal(mu, sigma^2) case, the variance of y-hat(mu) is s^2 and the variance of hat(mu)-mu is s^2/n.  The Fisher Information matrix is var of hat(mu)-mu (analogously).
* So notation is bad. type="states" implies xtT which is E[X(t)|Y(1)=y(1), hat(Theta)] but it's not that. Its sd of X(t)|Y(1)=y(1), hat(Theta).  So that interval is very much NOT a confidence interval (nor a prediction interval). It is just the quantiles of X(t)|Y(1)=y(1).
* Question is how do you use the FI to compute var of hat(E[X(t)])-E[X(t)]?

**Observations**

This needs lots of work. The y equation is different because it is = stoch var + error = f(X(t)) + v 

* *quantiles on fitted y* So we can talk about E_v(y|X(t)) having a distribution while E_v[E_X(f(X(t)+v)] is a point, i.e. integrate over both v and X.
* *quantiles on new y* We can also show quantiles on new y generated from the model, unconditioned on the data. Not prediction intervals since we did not include uncertainty in the parameters. Way too narrow for prediction intervals.
* *residuals variance* This is to evaluate residuals for outliers and estimate distribution of missing data.

**Comment by eeholmes** [comment]
After going around in circles, I settled on just adding ytT to tidy. I also decided that VtT is an estimate of the standard error of E[X(t)|data]-x(t), where x(t) is true state.  As you have more and more replicates of Y(t), VtT does shrink and xtT approaches x(t).  So VtT is behaving like a standard error where the thing being estimated is x(t) and its estimate is E[X(t)].  Not sure how to prove this, but since everything is normal, VtT should be an estimate of variance_Y|x E[X(t)|Y=y]-x(t).  So imagine many Y generated from Zx(t)+u+v(t) and then estimate E[X(t)]. The variance of E[X(t)]-x(t) should be VtT.

I started a document on intervals called Intervals.Rnw
